src/pnpm-lock.yaml

```diff
@@ -119,20 +119,20 @@ importers:
       '@prisma/sdk': 'workspace:*'
       '@prisma/studio': 0.296.0
       '@prisma/studio-server': 0.296.0
-      '@sindresorhus/slugify': ^1.1.0
-      '@timsuchanek/copy': ^1.4.5
+      '@sindresorhus/slugify': 1.1.0
+      '@timsuchanek/copy': 1.4.5
       '@types/debug': 4.1.5
       '@types/jest': 26.0.14
       '@types/pg': 7.14.5
       '@types/sqlite3': 3.1.6
-      '@types/string-hash': ^1.1.1
-      '@types/verror': ^1.10.4
+      '@types/string-hash': 1.1.1
+      '@types/verror': 1.10.4
       '@types/ws': 7.2.6
       '@typescript-eslint/eslint-plugin': 4.2.0
       '@typescript-eslint/parser': 4.2.0
-      checkpoint-client: 1.1.12
+      checkpoint-client: 1.1.13
       dotenv: 8.2.0
-      escape-string-regexp: ^4.0.0
+      escape-string-regexp: 4.0.0
       eslint: 7.10.0
       eslint-config-prettier: 6.12.0
       eslint-plugin-eslint-comments: 3.2.0
```

Comment:
Not sure why there are all these changes tbh. I just run `pnpm install checkpoint-client@1.1.13`